externs/browser/html5.js

```diff
@@ -2386,29 +2386,29 @@ MediaError.prototype.message;
 /**
  * The fetching process for the media resource was aborted by the user agent at
  * the user's request.
- * @type {number}
+ * @const {number}
  */
-MediaError.MEDIA_ERR_ABORTED;
+MediaError.MEDIA_ERR_ABORTED = 1;
```

Comment:
As Ben mentioned in #3293, we generally avoid putting values in the externs. Otherwise looks good.

Reply:
Ok - wasn't able to convince anyone that the code size optimization was worth including the values for constants so I have stripped them out and re-pushed the changeset.